routes/applications.js

```diff
@@ -14,16 +14,12 @@ router.get("/applications", async (req, res) => {
     }
 });
 
-
-
 router.post("/applications", async (req, res) => {
```

Comment:
Can we please change from "/applications" to "/create-application" or something like this? :) I just mean singular for more clarity